### ISSUE TITLE
Don't working with autohide taskbar

### ISSUE THREAD

**Issue by ramvyr** [ChrisAnd1998/TaskbarXI#81]
Hi. Thanks for apply!
I need transparent  and autohide taskbar, and option autohide only fullscreen applys.

**Comment by fredericocurti** [comment]
yeah, it would be nice to use autohide, bringing the taskbar up when the mouse approaches it
